index.html

```diff
@@ -716,8 +748,10 @@ <h1>Terminology</h1>
       <dt>
         <dfn>Virtual Thing</dfn>
       </dt>
-      <dd>An instance of a Thing that represents a Thing that is located
-        on another system component.</dd>
+      <dd>A <a>Service</a> that represents another <a>Thing</a>.
```

Comment:
or thing*s*?

Reply:
See above comments to Ben.    I plan to broaden the definition a bit/a lot and include this.  But this does not mean it's only a collection: a Virtual Thing is still a Service, so it's still a software entity with a network interface.  But yeah, there are use cases where we want to use a Virtual Thing to interface to multiple other Things.

My main concern right now is that my proposed extended definition of Virtual Thing is sounding more and more like "Intermediary" and I want to avoid having two terms for the same idea.  I will leave it alone for now, but options are: (1) restricting Intermediaries to one Exposer (2) making Virtual Things a kind of Intermediary, but also allowing Devices (e.g. physical Things) to be Intermediaries.  I'm leaning towards the second option.  I think it would be odd to call a physical device a "virtual" Thing.